packages/graphql/src/schema/get-obj-field-meta.ts

```diff
@@ -432,6 +432,12 @@ function getObjFieldMeta({
                                 }
                                 primitiveField.defaultValue = parseInt(value.value, 10);
                                 break;
+                            case "BigInt":
+                                if (value?.kind !== Kind.INT && value?.kind !== Kind.STRING) {
+                                    throw new Error(typeError);
+                                }
+                                primitiveField.defaultValue = parseInt(value.value, 10);
+                                break;
                             case "Float":
                                 if (value?.kind !== Kind.FLOAT) {
```

Comment:
Unrelated to this PR, but I believe here we should accept also `INT`
```suggestion
                                if (value?.kind !== Kind.FLOAT || value?.kind !== Kind.INT) {
```

Reply:
Unrelated, but just done and integration test added!